packages/components/src/components/CodeEditor/index.tsx

```diff
@@ -16,6 +16,7 @@ export type CodeEditorProps = {
   lineWrapping?: boolean;
   errors?: SqError[];
   sourceId?: string;
+  fontSize?: number;
```

Comment:
I think `size?: "normal" | "small"`, like we use in other components, would be better (editor size is not just about its font size, and I'm not even sure that modifying font size on top level makes sense), but nbd for now.